sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -32,6 +33,30 @@ class AdaptiveQueryExecSuite
 
   import testImplicits._
 
+  protected lazy val skewData1: DataFrame = {
+    val df1 =
+      spark
+        .range(0, 1000, 1, 10)
+        .selectExpr("id % 5 as key1", "id as value1").toDF()
+    df1.createOrReplaceTempView("skewData1")
+    df1
+  }
+
+  protected lazy val skewData2: DataFrame = {
+    val df2 =
+      spark
+        .range(0, 1000, 1, 10)
+        .selectExpr("id % 1 as key2", "id as value2").toDF()
+    df2.createOrReplaceTempView("skewData2")
+    df2
+  }
+
+  protected override def beforeAll(): Unit = {
+    super.beforeAll()
+    skewData1
```

Comment:
what's the point here? We can just make them `val` not `lazy val`

Reply:
If we only access the data by view name, we can just create these 2 temp views in `beforeAll` and drop them in `afterAll`.